doc/tutorial/hello_world.dox

```diff
@@ -0,0 +1,165 @@
+ * topics exactly or can use wildcards to match the topics.  If we set
+ * @c noWildCard to @c DPS_TRUE then only a subscription to @c a/b/c/d
+ * will receive this publication.  Since we set @c noWildCard to @c
+ * DPS_FALSE here, subscriptions to @c a/#, @c a/+/+/d, or similar
+ * variations will receive this publication.
+ *
+ * Both the publication's key identifier and acknowledgement handler
+ * are set to @c NULL here; they are covered in later sections.
+ *
+ * @see DPS_SetPublicationData(), DPS_GetPublicationData(),
+ *      DPS_PublicationGetNode(), DPS_PublicationGetUUID(),
+ *      DPS_PublicationGetSequenceNum()
+ *
+ * @subsection sending-a-publication Sending a publication
+ * @snippet tutorial.c Sending a publication
+ *
+ * Once created and initialized with a set of topics, application
+ * payloads may be sent.  Payload data is simply an array of bytes in
+ * DPS, no assumptions are made with regards to the payload format.
+ *
+ * In this example the @c ttl parameter is zero, indicating that the
+ * publication will be sent best-effort to all active subscribing
+ * nodes.  A non-zero ttl is referred to as a retained publication
+ * and is covered later.
+ *
+ * Recall also that a publisher may send application data multiple
+ * times, and each send increments the sequence number of the
+ * publication.
+ *
+ * @section subscribing Subscribing
+ *
+ * @subsection creating-a-subscription Creating a subscription
+ * @snippet tutorial.c Creating a subscription
+ *
+ * Each subscription in DPS is represented by a @c DPS_Subscription.
+ * In this example we are creating a subscription with one topic with
+ * no wildcards, @c a/b/c/d.
+ *
+ * Wildcards may be used to match a broader set of topics.  A @c +
+ * matches any single topic level, and a @c # matches all topic levels
+ * from that level on.  In this instance since the publisher is allowing
+ * wildcard matching, the subscriber could use either @c a/b/+/d or @c
+ * a/# (among others) as the topic and still receive the publication.
+ *
+ * A subscription may also be created with multiple topics.  The
+ * publication must include @em all of the topics to be received.
+ *
+ * @see DPS_SetSubscriptionData(), DPS_GetSubscriptionData(),
+ *      DPS_SubscriptionGetNode()
+ *
+ * @subsection receiving-a-publication Receiving a publication
+ * @snippet tutorial.c Subscribing
+ *
+ * Publications are received asynchronously.  The first step in
+ * receiving a publication is to provide the publication handler to
+ * DPS and start the subscription.  The publication handler will be
+ * called for each received publication.
+ *
+ * @note Each instance of @c DPS_Node runs in its own thread.  The
+ * publication handler is dispatched from this thread.
+ *
+ * @snippet tutorial.c Receiving a publication
+ *
+ * This publication handler exercises the APIs for retrieving the
+ * subscription and publication information.
+ *
+ * @section acknowledging Acknowledging
```

Comment:
I think this section on the Acknowledging helps answer my question about the send and forget in the above example.